greatnovels.sql

```diff
@@ -0,0 +1,152 @@
+DROP DATABASE greatnovels;
+DROP USER 'greatnovels'@'localhost';
+FLUSH PRIVILEGES;
```

Comment:
Be sure to delete debug code before finalizing a PR. These throw an error on a machine that doesn't already have the database setup.